docs/source/readers.rst

```diff
@@ -724,10 +724,13 @@ CMD Options                     JSON Options                 Definition
 ``--dbm-client-secret``         ``client_secret``            OAuth2 secret
 ``--dbm-access-token``          ``access_token``             (Optional) Access token for OAuth2
 ``--dbm-refresh-token``         ``refresh_token``            Refresh token for OAuth2
-``--dbm-query-request-type``    ``query_request_type``       Doubleclick Bid Manager API request type. Possible values: existing_query, custom_query, existing_query_report, custom_query_report, lineitems_objects, sdf_objects and list_reports.
+``--dbm-query-request-type``    ``query_request_type``       Doubleclick Bid Manager API request type. Possible values: existing_query, custom_query, existing_query_report, custom_query_report, custom_scheduled_query, lineitems_objects, sdf_objects and list_reports.
 ``--dbm-query-id``              ``query_id``                 Query ID.
 ``--dbm-query-title``           ``query_title``              Query title, used to name the reports generated from this query in DV360 UI.
 ``--dbm-query-frequency``       ``query_frequency``          How often the query is run. Possible values can be found `here <https://developers.google.com/bid-manager/v1/queries#schedule.frequency>`__. Default: ONE_TIME.
+``--dbm-query-timezone-code``   ``query_timezone_code``      Canonical timezone code for report generation time. Defaults to America/New_York.  
```

Comment:
en updatant la doc oublies pas de faire la doc côté json aussi qui est juste en dessous normalement

Reply:
it is already included here (column JSON options), or am I missing something ?

Reply:
ah yes my bad sorry